From f9ec1b4338d97ba4b16a0411e2a5d2a59c2f4e42 Mon Sep 17 00:00:00 2001
From: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Date: Thu, 25 Apr 2019 09:38:45 -0700
Subject: [PATCH] FROMLIST: Input: add KEY_KBD_LAYOUT_NEXT

The HID usage tables define a key to cycle through a set of keyboard
layouts, let's add corresponding keycode.

Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>

BUG=b:120873301
TEST=Build

(am from https://patchwork.kernel.org/patch/10917507/)
Signed-off-by: Dmitry Torokhov <dtor@chromium.org>
Change-Id: Ib55bb8f564960350d8e111ec0f49dbf2bebaf033
Reviewed-on: https://chromium-review.googlesource.com/1586635
Reviewed-by: Benson Leung <bleung@google.com>
Reviewed-by: Sean Paul <seanpaul@chromium.org>
Reviewed-by: Guenter Roeck <groeck@chromium.org>
---

diff --git a/include/uapi/linux/input-event-codes.h b/include/uapi/linux/input-event-codes.h
index 34e7b53..6774ebde 100644
--- a/include/uapi/linux/input-event-codes.h
+++ b/include/uapi/linux/input-event-codes.h
@@ -604,6 +604,7 @@
 #define KEY_SCREENSAVER		0x245	/* AL Screen Saver */
 #define KEY_VOICECOMMAND		0x246	/* Listening Voice Command */
 #define KEY_ASSISTANT		0x247	/* AL Context-aware desktop assistant */
+#define KEY_KBD_LAYOUT_NEXT	0x248	/* AC Next Keyboard Layout Select */
 
 #define KEY_BRIGHTNESS_MIN		0x250	/* Set Brightness to Minimum */
 #define KEY_BRIGHTNESS_MAX		0x251	/* Set Brightness to Maximum */
